round4/Team-4.md

```diff
@@ -0,0 +1,72 @@
+5. **Clean UI**: Our web blocker features an intuitive and neat interface, allowing you to easily manage block lists and set blocking that works for you.
+6. **Free**: Our web blocker is completely free, allowing users to focus solely on their web usage.
+7. **Performant**: Our web blocker is optimized to run smoothly without slowing down the user's browser.
+
+## User Stories
+
+1. As a student, I want to block social media websites during my study sessions to maintain focus and improve my academic performance.
+2. As a parent, I want to set up time limits and block inappropriate content on my child's devices to ensure a safe and productive online experience.
+3. As a remote worker, I want to block non-work-related websites during designated work hours to create a more disciplined and productive routine.
+4. As a gamer, I want to block gaming forums and websites during my work periods to minimize distractions and improve my academic performance.
+5. As a creative professional, I want to block self-critical or negative websites during my creative sessions to foster a more positive and productive mindset.
+6. As a digital minimalist, I want to block unnecessary websites and limit my online time to reduce digital clutter and increase mindfulness.
+7. As a meditation practitioner, I want to block news websites and social media platforms during my meditation sessions to create a serene and distraction-free environment.
+8. As a cybersecurity-conscious individual, I want to block suspicious or phishing websites to protect my personal information and enhance online security.
+9. As a recovering addict, I want to block websites related to addictive substances or behaviors to support my journey towards sobriety and wellness.
+10. As a financial planner, I want to block impulse-buying websites during my budgeting sessions to stick to my financial goals and avoid unnecessary expenses.
+11. As a researcher, I want to block social media and news websites during my research sessions to stay focused and avoid information overload.
+12. As a productivity coach, I want to recommend personalized website blocking settings to my clients based on their specific goals and challenges.
+13. As an individual looking to improve my mental health, I want to block triggering or distressing websites during vulnerable moments to protect my emotional well-being and promote self-care.
+14. As a developer, I require a feature in our productivity tool to block access to non-relevant websites during work hours, ensuring my focus remains on programming tasks and problem-solving without distractions.
+15. As a founder, I need a web blocking extension to help me stay focused on strategic tasks and minimize distractions during critical business operations.
+
+## Notes
+
+### Modes
+1. **À La Carte**: Set and name a time period and a set of block lists to prevent access to these websites during this time.
```

Comment:
I think this is a really both manageable and intuitive feature. It introduces the notion of "block list" which is a good design element and also intuitive.